src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -83,7 +83,26 @@ VALUES
     ('02438278-ab93-4689-b123-7ac4e78f59fe', 'smsWebEmail_1000'    , 'responses per site'           , '1000 SMS/web/email responses per site'),
     ('67e1c174-8443-4883-b51f-43c297fa9c08', 'smsWebEmail_1500'    , 'responses per site'           , '1500 SMS/web/email responses per site'),
     ('1a36f980-76a4-4b3d-b36d-46afc6655124', 'dataMigration'       , 'data migrations'              , 'per data migration'),
-    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service');
+    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service'),
+    ('f24afc45-971d-41fa-9beb-6f577e987c6d', 'patients_1-20k'      , 'patients'                     , 'per patient for 1-19,999 patients'),
+    ('0fb82859-aa6c-4db5-b513-c776c7278aca', 'patients_20-150k'    , 'patients'                     , 'per patient 20,000 - 149,999 patients'),
+    ('7b2c46d4-e5c1-4315-a05e-021e43d485ee', 'patients_150-250k'   , 'patients'                     , 'per patient 150,000 - 249,999 patients'),
+    ('d37f63e0-0bb2-4703-9f86-f22057231a00', 'patients_250k'       , 'patients'                     , 'per patient 250,000 patients & above'),
+    ('950365df-456d-4c72-bc6f-74a0f7535728', 'patientRegistered'   , 'patients'                     , 'per registered patient'),
+    ('2c25c251-168d-4f14-a637-3d09fe7440b7', 'onlineConsultationGP', 'consultations'                , 'per online consultation non GP practices'),
+    ('edb80481-d99a-4d25-8d58-da6881b453f7', 'videoConsultationGP' , 'consultations'                , 'per video consultation non GP practices'),
+    ('a2441010-cfd0-4a0a-bbf7-e73e279b4298', 'referralEps_1-5k'    , 'referral episodes'            , 'per referral episode 1-5000 episodes'),
```

Comment:
As above, we should use an en dash here, e.g. `'per referral episode 1–5000 episodes'`.

Reply:
Updated description to use en dash character.